<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dimming Debugger     DisConnect..</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>222.jpg</normaloff>222.jpg</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>371</width>
      <height>291</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>One Pattern</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_pattern">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>381</width>
        <height>261</height>
       </rect>
      </property>
      <property name="rowCount">
       <number>8</number>
      </property>
      <property name="columnCount">
       <number>8</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>45</number>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>21</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <item row="0" column="0">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="0" column="1">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="0" column="2">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="0" column="3">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="0" column="4">
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item row="0" column="5">
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item row="0" column="6">
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item row="0" column="7">
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item row="1" column="0">
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item row="1" column="1">
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item row="1" column="2">
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item row="1" column="3">
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item row="1" column="4">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="1" column="5">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="1" column="6">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="1" column="7">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="2" column="0">
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item row="2" column="1">
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item row="2" column="2">
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item row="2" column="3">
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item row="2" column="4">
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item row="2" column="5">
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item row="2" column="6">
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item row="2" column="7">
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item row="3" column="0">
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item row="3" column="1">
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item row="3" column="2">
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item row="3" column="3">
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item row="3" column="4">
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item row="3" column="5">
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item row="3" column="6">
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item row="3" column="7">
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item row="4" column="0">
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item row="4" column="1">
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item row="4" column="2">
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item row="4" column="3">
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item row="4" column="4">
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item row="4" column="5">
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item row="4" column="6">
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item row="4" column="7">
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item row="5" column="0">
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item row="5" column="1">
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item row="5" column="2">
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item row="5" column="3">
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item row="5" column="4">
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item row="5" column="5">
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item row="5" column="6">
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item row="5" column="7">
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item row="6" column="0">
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item row="6" column="1">
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item row="6" column="2">
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item row="6" column="3">
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item row="6" column="4">
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item row="6" column="5">
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item row="6" column="6">
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item row="6" column="7">
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item row="7" column="0">
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item row="7" column="1">
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item row="7" column="2">
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item row="7" column="3">
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item row="7" column="4">
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item row="7" column="5">
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item row="7" column="6">
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item row="7" column="7">
       <property name="text">
        <string>6</string>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_addpattern">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>AddPat</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_MemDataSelect">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_saveMemDataToFile">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>SaveToFile</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_prev">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>75</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>PREV</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_next">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>75</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>NEXT</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Pattern Index:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_readpattern">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ReadPat</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_loadMemDatafromFile">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>LoadfromFile</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>361</width>
      <height>441</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>470</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Curretn Pat delay time(second):</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_patdelays">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>470</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_delpattern">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>90</y>
      <width>111</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>DelPat</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sendpattochip">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>460</y>
      <width>111</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>SendToChip</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_savecurrentpat">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>130</y>
      <width>111</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>SaveCurPat</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_readfromchip">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>460</y>
      <width>131</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ReadFromChip</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_listpats">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>769</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="toolTip">
     <string extracomment="loading setting file or othes"/>
    </property>
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen_setting_file"/>
    <addaction name="actionSave_setting_file"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>Setting</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSetting"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_setting_file">
   <property name="text">
    <string>Open setting file</string>
   </property>
  </action>
  <action name="actionSave_setting_file">
   <property name="text">
    <string>Save setting file</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>quit</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
